foundation/foundation-mda/src/main/resources/templates/notifications/notification.configuration.store.tilt.vm

```diff
@@ -0,0 +1,14 @@
+project_path = os.path.abspath('.')
+# Update configuration_files_path to match the path of the config files to be loaded into the configuration store. Example 'src/main/resources/config'
```

Comment:
Q/A: This would be relative to the directory from which `tilt up` is run, right? I might suggest a path that makes that more clear like `my-project-deploy/src/main/resources/config`.